colorbleed/plugins/maya/load/load_camera.py

```diff
@@ -27,8 +27,15 @@ def process_reference(self, context, name, namespace, data):
                           returnNewNodes=True)
 
         cameras = cmds.ls(nodes, type="camera")
-        for camera in cameras:
-            cmds.camera(camera, edit=True, lockTransform=True)
+
+        # Check the Maya version, lockTransform has been introduced since
+        # Maya 2017
+        if cmds.about(version=True) >= "2018":
```

Comment:
Oof, the comparison on strings. Will we rely on this? I believe maya 2016 Extension 2 actually return "2016.5" which will be (in a string comparison) larger than "2018".

Should we force it to an integer?
```
    maya_version = int(cmds.about(version=True))
    if maya_version >= 2018:
        # do whateverrrrr
        pass
```